core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -44,7 +44,7 @@ object ManagedGroup {
 case class ManagedGroup(membersGroup: RawlsGroup, adminsGroup: RawlsGroup) extends Managed
 
 case class RawlsBillingAccount(accountName: RawlsBillingAccountName, firecloudHasAccess: Boolean, displayName: String)
-case class RawlsBillingProject(projectName: RawlsBillingProjectName, status: CreationStatuses.CreationStatus, billingAccount: Option[RawlsBillingAccountName], message: Option[String], cromwellBackend: Option[CromwellBackend] = None, servicePerimeter: Option[ServicePerimeterName] = None, googleProjectNumber: Option[GoogleProjectNumber] = None, invalidBillingAccount: Boolean = false, spendReportDataset: Option[String] = None, spendReportTable: Option[String] = None) {
+case class RawlsBillingProject(projectName: RawlsBillingProjectName, status: CreationStatuses.CreationStatus, billingAccount: Option[RawlsBillingAccountName], message: Option[String], cromwellBackend: Option[CromwellBackend] = None, servicePerimeter: Option[ServicePerimeterName] = None, googleProjectNumber: Option[GoogleProjectNumber] = None, invalidBillingAccount: Boolean = false, spendReportDataset: Option[String] = None, spendReportTable: Option[String] = None, spendReportDatasetGoogleProject: Option[String] = None) {
```

Comment:
I'm thinking I will add `BigQueryDatasetName` and `BigQueryTableName` case classes to workbench-libs and then refactor, deprecate, and version the existing methods in `GoogleBigQueryService` so that they'll use these new case classes as well